sdk/redis/arm-rediscache/src/static-helpers/serialization/serialize-record.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function serializeRecord(item: any, excludes?: string[], serializer?: (item: any) => any) {
+  excludes = excludes ?? [];
+  const res: any = {};
+  for (const key of Object.keys(item)) {
```

Comment:
if item===undefined or null, get error:
`Cannot convert undefined or null to object `